ush/python/pygfs/task/aero_prepobs.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+import gzip
+import tarfile
+import re
+from logging import getLogger
+from typing import List, Dict, Any, Union
+
+from wxflow import (AttrDict, FileHandler, rm_p, rmdir,
+                    Task, add_to_datetime, to_timedelta, to_datetime,
+                    datetime_to_YMD,
+                    chdir, Executable, WorkflowException,
+                    parse_j2yaml, save_as_yaml, logit)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class AerosolObsPrep(Task):
+    """
+    Class for preparing and managing aerosol observations
+    """
+    def __init__(self, config: Dict[str, Any]) -> None:
+        super().__init__(config)
+
+        _window_begin = add_to_datetime(self.runtime_config.current_cycle, -to_timedelta(f"{self.config['assim_freq']}H") / 2)
+        _window_end = add_to_datetime(self.runtime_config.current_cycle, +to_timedelta(f"{self.config['assim_freq']}H") / 2)
+
+        local_dict = AttrDict(
+            {
+                'window_begin': _window_begin,
+                'window_end': _window_end,
+                'sensors': str(self.config['SENSORS']).split(','),
+                'data_dir': self.config['VIIRS_DATA_DIR'],
+                'input_files': '',
+                'OPREFIX': f"{self.runtime_config.CDUMP}.t{self.runtime_config.cyc:02d}z.",
+                'APREFIX': f"{self.runtime_config.CDUMP}.t{self.runtime_config.cyc:02d}z."
```

Comment:
This should probably be RUN:
```suggestion
               'OPREFIX': f"{self.runtime_config.RUN}.t{self.runtime_config.cyc:02d}z.",
                'APREFIX': f"{self.runtime_config.RUN}.t{self.runtime_config.cyc:02d}z."
```